EventListeners.js

```diff
@@ -0,0 +1,53 @@
+// Why does this bring an error?
```

Comment:
It might be because using import or export makes the JS file become what's known as a module, and to properly use a JS module you gotta set the script tag like <script type="module" src="my-file.js">

This might be a good read: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#introducing_an_example

Reply:
should I change the html so that it has the script type to have type module for this file then?

Reply:
Yeah you could to see if it solves the problem. Though we still need to properly set up the overall load order of things from the HTML file, so that's likely to change soon.